contribute.rst

```diff
@@ -28,34 +28,34 @@ Generally speaking we expect all contributions to adhere to the following conven
 
 -  Robustly tested code:
 
-   -  All core functionality unit tested with `py.test <https://docs.pytest.org/en/latest/>`__ and edge cases considered. We strongly suggest you use Test Driven Development. We also require details and evidence of any manual testing to show no existing functionality is unexpectedly broken.
+   -  All core functionality is unit tested with `py.test <https://docs.pytest.org/en/latest/>`__ and edge cases considered. We strongly suggest you use Test Driven Development. We also require details and evidence of any manual testing to show no existing functionality is unexpectedly broken.
 
 -  Documented code:
 
-   -  We expect a clear docstring per module, class, and function, explaining what it does at a minimum. This is to reduce developer onboarding time and reduce barriers to entry for new developers who want to get involved. They should be formatted as per `Google-style Sphinx Docstrings <http://www.sphinx-doc.org/en/stable/ext/example_google.html>`__.
+   -  We expect a clear docstring per module, class, and function, explaining what it does at a minimum. This is to reduce both developer onboarding time and the barriers to entry for new developers who want to get involved. They should be formatted as per `Google-style Sphinx Docstrings <http://www.sphinx-doc.org/en/stable/ext/example_google.html>`__.
 
 -  No code smells:
 
    -  Don’t Repeat Yourself, keep your code DRY.
 
-   -  Avoid excessive use of conditional statements, your functions should be doing the minimum possible for maximum effect. Think about `polymorphism <https://www.digitalocean.com/community/tutorials/how-to-apply-polymorphism-to-classes-in-python-3>`__ and the `Single Responsibility Princible (SRP) <https://robots.thoughtbot.com/back-to-basics-solid#single-responsibility-principle>`__.
+   -  Avoid excessive use of conditional statements, your functions should be doing the minimum possible for maximum effect. Think about `polymorphism <https://www.digitalocean.com/community/tutorials/how-to-apply-polymorphism-to-classes-in-python-3>`__ and the `Single Responsibility Principle (SRP) <https://robots.thoughtbot.com/back-to-basics-solid#single-responsibility-principle>`__.
```

Comment:
A comma doesn't seem like quite the right punctuation in the middle of this sentence.